src/index.html

```diff
@@ -29,40 +30,44 @@
   <link href="/assets/images/favicon_16x16.png" rel="icon" sizes="16x16" type="image/png">
   <link href="/assets/images/favicon_32x32.png" rel="icon" sizes="32x32" type="image/png">
 
-  <noscript><meta http-equiv="Refresh" content="0; URL=/enable-javascript.html"></noscript>
+  <noscript>
+    <meta http-equiv="Refresh" content="0; URL=/enable-javascript.html"></noscript>
 
   <base href="/">
 
-  <link href="https://fonts.googleapis.com/css?family=Lato:300,300i,400&amp;subset=latin-ext"
-        rel="stylesheet"
-        data-ng-if="isBcourses">
+  <link href="https://fonts.googleapis.com/css?family=Lato:300,300i,400&amp;subset=latin-ext" rel="stylesheet"
+    data-ng-if="isBcourses">
```

Comment:
@nkutub - Maybe we should get rid of this line given that we don't support bCourses / Canvas?

Reply:
removed